From e3ebc237397387251828e52f4c21509d977e7797 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 13 Nov 2019 16:47:47 +0800
Subject: [PATCH] drm: bridge: cadence: hdmi: remove video mode limition

combine mode is supported by imx8qm DPU.
imx8qm HDMI could support full feature.
Remove the 1080p60 limition.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
---
 drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
@@ -305,8 +305,8 @@ cdns_hdmi_bridge_mode_valid(struct drm_b
 			mode->flags & DRM_MODE_FLAG_INTERLACE)
 		return MODE_BAD;
 
-	/* MAX support pixel clock rate 148.5MHz */
-	if (mode->clock > 148500)
+	/* MAX support pixel clock rate 594MHz */
+	if (mode->clock > 594000)
 		return MODE_CLOCK_HIGH;
 
 	/* 4096x2160 is not supported */
